streams/src/main/java/org/apache/kafka/streams/KafkaClientSupplier.java

```diff
@@ -63,4 +64,13 @@
      * @return an instance of Kafka consumer
      */
     Consumer<byte[], byte[]> getRestoreConsumer(final Map<String, Object> config);
+
+    /**
+     * Create a {@link Consumer} which is used to restore records for {@link GlobalKTable}.
```

Comment:
Nit: `restore` -> `consume` ? This consumer is used for regular processing -- "restore" sounds like fault-tolerance only.